app/com/linkedin/drelephant/tuning/engine/SparkHBTParamRecommender.java

```diff
@@ -360,4 +430,62 @@ private int getMemoryIncreaseForGC() {
       return 0;
     }
   }
+
+  /**
+   * This method suggest three parameters : spark.dynamicAllocation.minExecutors, spark.dynamicAllocation.maxExecutors
+   *  and spark.executor.instances
+   * For now we are changing these parameters only in case we are changing number of executor core
+   */
+  private void suggestNumExecutorParameters() {
+    if (suggestedCore != lastRunExecutorCore) {
+      if (lastRunDynamicAllocationMinExecutors != null) {
+        suggestedDynamicAllocationMinExecutors =
+            (int) Math.ceil(lastRunDynamicAllocationMinExecutors * lastRunExecutorCore / suggestedCore);
```

Comment:
What if min executor config is 0? Can it be?

Reply:
Not sure about this. I suppose it cannot be 0. Now we are not looking into min executor anyway.